Sources/Capabilities/Drawing/MediaEditorAnnotationView.swift

```diff
@@ -0,0 +1,230 @@
+import UIKit
+import AVFoundation
+import PencilKit
+
+@available(iOS 13.0, *)
+protocol MediaEditorAnnotationViewUndoObserver: NSObject {
+    func mediaEditorAnnotationView(_ annotationView: MediaEditorAnnotationView, isHidingUndoControls: Bool)
+    func mediaEditorAnnotationViewUndoStatusDidChange(_ view: MediaEditorAnnotationView)
+}
+
+/// Wrapper view that contains an image view and a PencilKit canvas to allow
+/// drawing on top of the image.
+///
+@available(iOS 13.0, *)
+class MediaEditorAnnotationView: UIView {
+
+    private let imageView = UIImageView()
+    private let canvasView = PKCanvasView()
+
+    private var bottomConstraint: NSLayoutConstraint!
+
+    weak var undoObserver: MediaEditorAnnotationViewUndoObserver? = nil
```

Comment:
Do we need to explicitly assign `nil` to `undoObserver` for any special reason?

Reply:
Nope! I'll remove it :)